src/lib/appsRegistry.js

```diff
@@ -1,14 +1,21 @@
+import { inRange } from 'lodash-es';
 import blockchainExplorerIcon from '../assets/icons/aepps/blockchain-explorer.svg';
 import tokenMigrationIcon from '../assets/icons/aepps/token-migration.jpg';
 
 export { default as DEFAULT_ICON } from '../assets/icons/aepps/default.svg';
 
-export const aeternityApps = [{
+const showVotingApp = inRange(
+  Date.now(),
+  (new Date('2019-05-06T22:00:00.000Z')).getTime(),
+  (new Date('2019-05-14T22:00:00.000Z')).getTime(),
```

Comment:
would be okay to have it shown afterwards as well, no need to remove it I guess

Reply:
Also is this timezone agnostic?

Reply:
Firstly I have took ISO dates with timezones: `2019-05-07T00:00:00+0200` but appears that Date in Safari can't parse them, so I converted them to usual ISO string by `(new Date('2019-05-07T00:00:00+0200')).toISOString()` in Chrome and used them instead. ISO date strings are relative to UTC, so as I understand they are timezone agnostic.

Reply:
maybe using unix timestamp would be saver and easier to work with